drivers/src/persistent.rs

```diff
@@ -52,6 +55,70 @@ pub struct IdevIdCsr {
     csr: [u8; MAX_CSR_SIZE],
 }
 
+#[cfg(feature = "fmc")]
```

Comment:
What does this "fmc" feature mean? It looks like it is set when building runtime (which makes sense since these functions seem needed). But I'm not quite understanding the intent from the name.